creator-node/src/index.js

```diff
@@ -96,11 +105,12 @@ const startApp = async () => {
 
     /** Run app */
     await BlacklistManager.blacklist(ipfs)
+    await BlacklistManager.blacklist(ipfsLatest)
```

Comment:
why is this necessary?

Reply:
whoops
it isn't bc they both point to the same ipfs node but with different signature calls
in the future, we will still have 1 blacklist call, but to the newer version of API
removing :)